mne/utils/docs.py

```diff
@@ -2499,13 +2499,31 @@
     If it is a 2-tuple of minimum and maximum limit, then those
     physical ranges will be used. Only used for exporting EDF files.
 """
+docdict['export_params_add_ch_type'] = """
+add_ch_type : bool
+    Whether to add the channel type of the channel currently set
+    (default is False) to the signal label. Only used for
+    exporting EDF files.
+"""
 docdict['export_eeglab_note'] = """
 For EEGLAB exports, channel locations are expanded to full EEGLAB format.
 For more details see :func:`eeglabio.utils.cart_to_eeglab`.
 """
 docdict['export_edf_note'] = """
-For EDF exports, only EEG, ECoG and sEEG data are supported. In
-addition, EDF does not support storing a montage.
+For EDF exports, only channels that are measured in terms of Voltage are
+supported. For example, 'eeg', 'ecog', 'seeg', 'bio', 'emg', 'eog',
+'ecg', 'dbs', 'bio' data are supported. However, EDF files do not
+require the storage of channel types, so when re-reading the file
```

Comment:
Maybe I missed something in the earlier discussion, but here you say "EDF does not require storing channel type".  That implies that it's possible, which means we can (and should) store channel type when we export.  Further down you give an example of "misc" being a problem, but "misc" channels are often in Volts (as are "stim", which are not included in your list of good types).  So reading this note, it's not clear to me exactly what situations will lead to channel type information being lost on round-trip.  Can you clarify what exactly is the issue, and make the note text here reflect that more closely?

Is the problem that *only a predefined lexicon of channel type names* are allowed?  Or is it that EDF does not even *allow* storing channel types (other than as part of the channel label)?  Or something else?

As appropriate, please distinguish between problems that are limitations of the EDF format, and problems due to other EDF writers that may not include all optional information.  (The latter case doesn't need to be addressed in this docstring, since this is the docstring for our *exporter*...  problems related to reading other writers' EDF files belong in the docstring for our EDF *reader*).

Reply:
I updated the docs significantly to reflect what are the limitations of exporting and reading EDF files. 

The problem for writing/reading channel types is that there is no predefined lexicon of channel type names. Therefore, there are just commonly "suggested ones", such as EEG. 

lmk wyt?